static/js/components/EducationDialog.js

```diff
@@ -23,11 +26,16 @@ export default class EducationDialog extends ProfileFormFields {
   }
   educationLevelLabels: Object;
 
-  static propTypes = {
-    open:           React.PropTypes.bool,
-    onClose:        React.PropTypes.func,
-    onSave:         React.PropTypes.func,
-    showLevelForm:  React.PropTypes.bool,
+  props: {
+    ui:                           UIState,
+    setEducationDialogVisibility: () => void,
+    setEducationDegreeLevel:      () => void,
+    setEducationDialogIndex:      () => void,
+    clearProfileEdit:             () => void,
+    saveProfile:                  SaveProfile,
```

Comment:
again, i think it would be a lot more helpful if this type name was `SaveProfileFunction`. it's a more accurate description of the type we're expecting, and it avoids potential confusion with the `Profile` type. same comment goes for `UpdateProfile`